content/help/ops/security/keys-and-certs/index.md

```diff
@@ -17,7 +17,33 @@ istio.my-sa             istio.io/key-and-cert               3      24d
 
 Where `my-ns` and `my-sa` are the namespace and service account your pod is running as.
 
-Then make sure the certificate is valid with:
+The pods can be listed by the command:
```

Comment:
Maybe remove the instructions for listing pods? It doesn't seem to be needed here.

Reply:
Done.